docs/RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Prepare release
```

Comment:
Is this a step by step process of actions, or is this a checklist?

passing tests is not an activity for example.

Reply:
You are right that I have not made a distinction between steps and gating items. Will fix.